x-pack/plugins/cases/common/api/metrics/case.ts

```diff
@@ -73,10 +73,6 @@ const AlertUsersMetricsRt = rt.strict({
 });
 
 export const SingleCaseMetricsRequestRt = rt.strict({
-  /**
-   * The ID of the case.
-   */
-  caseId: rt.string,
```

Comment:
I'm removing this to make moving the decode into the `try/catch` easier. We don't really need this as part of the type because it's part of the request params not the query.